Nikita_Konkov_Application/Selenium_Basics/Selenium_Basics/EpamHowWeDoItOurWorkNavigationTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Selenium_Basics
+{
+    [TestFixture]
+    public class EpamHowWeDoItOurWorkNavigationTest
```

Comment:
Is it navigation test? Why do you created separate test class for each test?

Reply:
Corrected title of the class. I created them separated because it's more comfortable to me. I've united them already.